README.md

```diff
@@ -1278,3 +1277,28 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+ts other than the default export are then ignored by this package's structure loader.
```

Comment:
was something moved?

Reply:
my bad the last paragraph (Licence) was duplicated by mistake. I have fixed that with new commit